dizoo/atari/config/serial/pong/pong_trex_offppo_config.py

```diff
@@ -0,0 +1,77 @@
+from copy import deepcopy
+from easydict import EasyDict
+
+pong_ppo_config = dict(
+    exp_name='pong_trex_offppo',
+    env=dict(
+        collector_env_num=16,
+        evaluator_env_num=8,
+        n_evaluator_episode=8,
+        stop_value=20,
+        env_id='PongNoFrameskip-v4',
+        frame_stack=4,
+        manager=dict(shared_memory=False, )
+    ),
+    reward_model=dict(
+    type='trex',
+    algo_for_model = 'ppo',
+    env_id='PongNoFrameskip-v4',
+    input_size=5,
+    hidden_size=64,
+    batch_size=64,
+    learning_rate=1e-5,
+    update_per_collect=1,
+    expert_model_path='/Users/nieyunpeng/Documents/open-sourced-algorithms/TREX/dizoo/atari/config/serial/pong/ppo_offpolicy_pong',
```

Comment:
don't add personal information in public code

Reply:
Thanks for the comment! 

As I have written above in Description, I will remove all the personal information before merging.